src/PackageCompiler.jl

```diff
@@ -483,6 +483,8 @@ function create_sysimage(packages::Union{Nothing, Symbol, Vector{String}, Vector
                          compat_level::String="major",
                          extra_precompiles::String = "",
                          )
+    # We call this at the very beginning to make sure that the user has a compiler available. Therefore, if no compiler is fiund, we throw an error immediately, instead of making the user wait a while before the error is thrown.
```

Comment:
typo on found and linewidth
```suggestion
    # We call this at the very beginning to make sure that the user has a compiler available.  Therefore, if no compiler 
    # is found, we throw an error immediately, instead of making the user wait a while before the error is thrown.
```